site/package.json

```diff
@@ -44,5 +44,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "prettier": "^2.7.1"
```

Comment:
we have already got one in /root/package.json, can we merge and only keep one?
https://github.com/opensquare-network/statescan-v2/blob/main/package.json#L13